portfolio/src/main/webapp/map-script.js

```diff
@@ -0,0 +1,157 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var map;
+var journeyCoordinates = [];
+
+/**
+ * Initial the map and add it to the map page.
+ */
+function initMap() {
+    var mapCenter = { lat: 30.168979, lng: -171.859749 }; // Somewhere in North Pacific Ocean
+    var myHometown  = { lat: 34.724384, lng: 113.640486 }; // Zhengzhou
+    var myArrival = { lat: 42.361145, lng: -71.057083 } // Boston
+    var myHighschool = { lat: 41.342082, lng: -72.913567 }; // Hamden
+    var myLocation = { lat: 40.678038, lng: -73.950493 }; // Brooklyn
+    var winterTravel = { lat: 34.050925, lng: -118.243498 }; // LA
+    var summerTravel = { lat: 31.208968, lng: 121.468941 }; // Shanghai
+    
+    journeyCoordinates = [ myHometown, myHighschool, myArrival, summerTravel, winterTravel, myLocation ];
+    var journeyPath = new google.maps.Polyline({
+        path: journeyCoordinates,
+        geodestic: true,
+        strokeColor: '#C1328E',
+        strokeOpacity: 1.0,
+        strokeWeight: 1,
+    });
+
+    map = new google.maps.Map(
+        document.getElementById('map'), 
+        {center: mapCenter,
```

Comment:
```suggestion
        {
          center: mapCenter,
```

Note that this might require adjusting the indentation of the lines from below.

Reply:
The indentation is added to the lines.